### ISSUE TITLE
SmartBanner.js loads at the end of the DOM and creates incorrect tab order

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15347]
## Describe the defect
This is an issue to track a known issue for SmartBanner.js, which it has not yet been determined for sure if we will use or not - if we end up not using it, we can close this issue.

We are considering using SmartBanner.js to expand the Mobile app banner to display in other non-Safari browsers as well as on Android devices. However, after testing I noticed that this loads towards the end of the DOM, after the page content which means the tab order is unexpected.

I would expect to have the mobile app banner be the first elements in the tab order, before even the header as it is displayed visually before the header. However, because of where it is in the DOM, it is actually the last tabbable element, after the footer.

## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by laflannery** [comment]
@jilladams @FranECross Should we close this now based on the discovery work done by @chriskim2311 #15339 as he made a PoC that resolved this issue?

**Comment by jilladams** [comment]
Yes! Closing.
